python-client-migration/README.md

```diff
@@ -0,0 +1,31 @@
+# Python OpenSearch Migration Examples
+In order to run those examples locally, install the dependencies:
+
+```shell
+pip install -r requirements.txt
+```
+
+Testing show that this is a four (4) lines change. Change the `import` statement, and the Python client for ES (Elasticsearch) and OS (OpenSearch).
+
+Changes related to `import`:
+
+```python
+- from elasticsearch import Elasticsearch
++ from opensearchpy import OpenSearch
+```
+
+Changes related to clients and arguments:
+
+```python
++ opensearch_client = Elasticsearch(ES_SERVICE_URI, use_ssl=True)
+- opensearch_client = OpenSearch(OS_SERVICE_URI, use_ssl=True)
+```
+
+See this commit diff as an example change: [python-client-migration/python_client_migration/main.py](https://github.com/aiven/opensearch-migration-examples/commit/58d35ad29ce0ba8337a383fbfe920f1644a80361)
+
+
```

Comment:
I think it might be worth adapting the text from the PR conversation - something like:
```
## How we made this example

* We started with a full ES example duplicated in es-main.py and main.py
* We then created a full OS example in os-main.py
* and then modified main.py to use the OS client with minimal changes.
```

Reply:
Great! I think it can be useful but I am thinking more this should be in [CONTRIBUTING.rst](https://github.com/aiven/opensearch-migration-examples/pull/10). What do you think? This is the proper process for all the examples afaik.

Reply:
I do like the idea of keeping the README  nice and simple and clear, as it was.
So that sounds sensible to me. Thank you.